src/ZopeUndo/tests/test_prefix.py

```diff
@@ -23,6 +23,8 @@ def test(self):
         p1 = Prefix("/a/b")
         for equal in ("/a/b", "/a/b/c", "/a/b/c/d"):
             self.assertEqual(p1, equal)
+        for equal in ("/a/b", "/a/b/c", "/a/b/c/d"):
+            self.assertEqual(p1 != equal, False)
```

Comment:
This is hard to read. Did you mean: ``self.assertNotEqual(p1, equal)``?

Reply:
Or maybe `self.assertFalse(p1 != equal)`? But even this would need a comment describing that `__ne__` has to be triggered by the test.

Reply:
@icemac @hannosch changed with assertFalse/assertTrue. Absolutely better to read. Thx.